driver/mobile/keyboard.go

```diff
@@ -0,0 +1,26 @@
+package mobile
+
+import (
+	"fyne.io/fyne"
+)
+
+// Keyboard represents a standard fyne keyboard
+type Keyboard int32
+
+const (
+	// DefaultKeyboard is the keyboard with default input style and "Done" return key
+	DefaultKeyboard Keyboard = iota
+	// MultiLineKeyboard is the keyboard with default input style and "return" return key
+	MultiLineKeyboard
+	// NumberKeyboard is the keyboard with number input style and "Done" return key
+	NumberKeyboard
+	// WebKeyboard is the keyboard with web input style and "Done" return key
+	WebKeyboard
```

Comment:
Is web keyboard for URLs or emails? this feels a little ambiguous.

Reply:
In code I chose hints for URL. But in manual tests with two keyboards they doesn't change at all. Maybe I did something wrong. Or at least on iOS it will be different.

Reply:
We could remove `WebKeyboard` as I think the most important types for now is one/multi line and number keyboards.

Reply:
Remember that when designing Fyne APIs we need to consider the semantic meaning here - if an OS chooses to display the same keyboard for URL and for Email is fine, but we know they are different and so present that in the API.